### PR TITLE
Add skylark macro for import to java

### PR DESCRIPTION
**Pull request by jcoveney**
closes https://github.com/bazelbuild/rules_scala/issues/18

### CODE REVIEW

**Comment by johnynek** [comment]
note @jcoveney is at @stripe which has signed a CLA.

Thanks sir!

File: scala/scala.bzl [hunk 1]
```diff
@@ -390,3 +390,15 @@ def scala_repositories():
...
+def scala_export_to_java(name, exports, runtime_deps):
+  jars = []
+  for target in exports:
+    jars.append("{}_deploy.jar".format(target))
+
+  native.java_import(
+    name = name,
+    # these are the outputs of the scala_library targets
+    jars = jars,
+    runtime_deps = ["org_scala_lang__scala_library"] + runtime_deps
```
Comment (johnynek) [inline scala/scala.bzl]: need to point to `@scala//:lib/...` I think the above is local to the tests here.

Comment (johnynek) [inline scala/scala.bzl]: note this `org_scala_lang_...` target is defined in the tests:

https://github.com/bazelbuild/rules_scala/pull/35/files#diff-c40927b6657800717d4b0b498e1bad55R42

File: test/BUILD [hunk 2]
```diff
@@ -23,6 +28,16 @@ java_import(
     runtime_deps = ["org_scala_lang__scala_library", "OtherJavaLib"]
 )
 
+scala_export_to_java(
+    name = "lib_import_2",
```
Comment (johnynek) [inline test/BUILD]: can we exercise this? Like make a binary that runs or something?

Comment (jcoveney) [inline test/BUILD]: added

File: scala/scala.bzl (hunk 1 above)
Comment (jcoveney) [inline scala/scala.bzl]: fixed

**Comment by jcoveney** [comment]
Updated!

**Pull request merged**